indico/modules/users/controllers.py

```diff
@@ -738,6 +740,24 @@ def _process_pending_users(self, results):
             if ext_id is not None:
                 cache.set(ext_id, self.externals[ext_id], timeout=86400)
 
+    @use_kwargs({
+        'category_id': fields.Integer(validate=lambda x: x >= 0),
+        'event_id': fields.Integer(validate=lambda x: x >= 0)
+    }, location='query')
+    def _process_args(self, event_id=None, category_id=None):
+        print(event_id, category_id)
+        super(RHProtected, self)._process_args()
```

Comment:
I'd keep the old-style `RHProtected._process_args(self)` call here; we have some places where we are explicit about which parent class methods we call (usually w/ protection checks though) and AFAIK it's not recommended to mix super and non-super parent calls.